### DISCUSSION TITLE
Compatible with existing code?

### DISCUSSION THREAD

**Discussion by jerone** [Giorgi/EntityFramework.Exceptions#64] [Q&A] [answered]
Does this package supports existing code where `DbUpdateException` exceptions are already caught?

For example, if my existing code looks like:
```csharp
try
{
    context.SaveChanges();
}
catch (DbUpdateException e) // <--
{
}
```
Will it catch the exceptions from this package?

**Comment by Giorgi** [comment, marked as answer]
All the exceptions defined in this package inherit from `DbUpdateException` so any existing code that catches them will also catch exceptions from this package.

**Reply by jerone** [reply]
Thank you for the answer. I created #65 with this answer.
